vault/auth.go

```diff
@@ -159,13 +160,25 @@ func (v *Vault) UserPassLogin() string {
 func (v *Vault) EC2Login() string {
 	role := env.Getenv("VAULT_AUTH_AWS_ROLE")
 	mount := env.Getenv("VAULT_AUTH_AWS_MOUNT", "aws")
+	nonce := env.Getenv("VAULT_AUTH_AWS_NONCE")
+	output := env.Getenv("VAULT_AUTH_AWS_NONCE_OUTPUT")
```

Comment:
I wonder if maybe naming this `VAULT_AUTH_AWS_NONCE_FILE` would be better, and making it possible to provide a nonce via a file (the case where the file exists already), or save it to that path if it's nonexistant.

Reply:
That should already work as it should be using the env lookup with _FILE support

Reply:
Changed